pkgs/top-level/darwin-packages.nix

```diff
@@ -101,6 +101,7 @@ makeScopeWithSplicing' {
     "mDNSResponder"
     "objc4"
     "ppp"
+    "xnu"
```

Comment:
It doesn’t, which is a reason to get rid of it. I don’t want to have to keep maintaining this.

Reply:
Maybe the `README` in `mkStub` should be generalized since it’s not just being used for frameworks that are present in the SDK? Though I doubt anyone will read it before we put in eval warnings anyway.